guides/common/modules/proc_synchronizing-python-repositories.adoc

```diff
@@ -13,17 +13,16 @@ You can view the Python packages in the {ProjectWebUI} at *Content* > *Content T
 +
 The *Label* field is populated automatically based on the name.
 . From the *Type* list, select *python*.
-. In the *Upstream URL* field, enter the URL for the upstream content source, for example `\https://pypi.org`.
+. In the *Upstream URL* field, enter the URL for the upstream content source, for example `https://pypi.org`.
+. Optional: Unselect *Verify SSL* if you do not want {ProjectServer} to verify the SSL certificate of the upstream content source.
```

Comment:
```suggestion
. Optional: Clear *Verify SSL* if you do not want {ProjectServer} to verify the SSL certificate of the upstream content source.
```
If this is a checkbox, the verb is 'clear'.

Reply:
Similar to @apinnick's comment above, this might be a case of overdocumenting the UI. I'm looking at the form in the web UI and the information on line 17 is already documented there.